api/app/invitationletter/generator.py

```diff
@@ -1,9 +1,91 @@
+from app.utils.emailer import send_mail
+from mailmerge import MailMerge
+from app import LOGGER
+import sys
+import os
+import requests
+from app.utils import emailer
+from app.utils import pdfconvertor
+from app.events.models import Event
+from app import db
+
+OFFER_EMAIL_BODY = """
+Dear {user_title} {first_name} {last_name},
+
+Congratulations! You've been selected to attend the {event_name}!
+
+Please see the attached document below for your acceptance of offer: {host}/offer
```

Comment:
@mthembbu I remember us discussing what this email copy should be. WOuld you update?

Reply:
Will do.